features/dom-geometry.yml

```diff
@@ -1,6 +1,7 @@
 name: DOM Geometry
 description: The `DOMMatrix`, `DOMPoint`, `DOMQuad` and `DOMRect` interfaces offer a way to represent points, rectangles, quadrilaterals and transformation matrices within JavaScript. They can be used in transformations in CSS, `<canvas>`, and SVG.
 spec: https://drafts.fxtf.org/geometry-1/
+group: dom
```

Comment:
I don't think this fits the description of the group. These geometry interfaces don't interact with the tree of nodes really. If I had to guess, I think they're just prefixed with "DOM" because the shorter names were already used in the wild for other things.

Reply:
Dropped with d6c38814a3c4abda69fe7971524da3a063a475b5.